examples/StreetlightsAPI/Program.cs

```diff
@@ -64,8 +64,6 @@ public void ConfigureServices(IServiceCollection services)
                         ["mosquitto"] = new Server("test.mosquitto.org", "mqtt")
                     }
                 };
-
-                options.JsonSchemaGeneratorSettings.SerializerSettings.Formatting = Formatting.Indented;
```

Comment:
The SerializerSettings should only be used as the contract for schema generation, not the serialization of the asyncapi document itself (these are two different things).